contrib/inventory/cobbler.py

```diff
@@ -215,7 +215,7 @@ def update_cache(self):
                 for (iname, ivalue) in iteritems(interfaces):
                     if ivalue['management'] or not ivalue['static']:
                         this_dns_name = ivalue.get('dns_name', None)
-                        if this_dns_name is not None and this_dns_name is not "":
+                        if this_dns_name != None and this_dns_name != "":
```

Comment:
The first should still be `is not`, the second should be `!=`.

See the pylint `singleton-comparison (C0121)` rule.  This is actually causing tests to fail on this PR currently:

https://app.shippable.com/github/ansible/ansible/runs/156289/3/tests